easybuild/easyconfigs/n/NEST/NEST-2.12.0-foss-2015b.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = 'CMakeMake'
+
+name = 'NEST'
+version = '2.12.0'
+
+homepage = 'http://www.nest-simulator.org/'
+description = """NEST is a simulator for spiking neural network models
+ that focuses on the dynamics, size and structure of neural systems rather
+ than on the exact morphology of individual neurons."""
+
+toolchain = {'name': 'foss', 'version': '2015b'}
+toolchainopts = {'optarch': True, 'pic': True}
```

Comment:
`optarch` is enabled by default, no need to specify it (but keep the `pic` obviously)